packages/nextjs/app/builders/0x7f6Ba565e35BB23C307992feBD841D5b2d8c8011/page.tsx

```diff
@@ -0,0 +1,117 @@
+import Link from "next/link";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const Home: NextPage = () => {
+  return (
+    <>
+      <div className="flex items-center flex-col grow pt-10">
+        <div className="px-5 justify-center items-center flex flex-col bg-base-100 w-full max-w-3xl rounded-3xl">
+          {/* 添加头像 */}
+          <img
+            src="https://avatars.githubusercontent.com/u/18190481?s=400&u=43a299844e466375bb4fe2235125925bff43b9ff&v=4"
+            alt="Avatar"
+            className="rounded-full w-32 h-32 mb-4"
+          />
```

Comment:
Consider using Next.js's `<Image>` component for better performance and built-in optimizations like lazy loading and responsive sizing.